releases/latest/beta/Dockerfile.ubuntu.adoptopenjdk11

```diff
@@ -1,7 +1,7 @@
 FROM adoptopenjdk:11-jre-openj9
 ARG LIBERTY_VERSION=20.0.0.12
```

Comment:
beta version is 21.0.0.1 : https://repo1.maven.org/maven2/io/openliberty/beta/openliberty-runtime/21.0.0.1-beta/